test/functional/ex_cmds/mkview_spec.lua

```diff
@@ -0,0 +1,67 @@
+local lfs = require('lfs')
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local get_pathsep = helpers.get_pathsep
+local eq = helpers.eq
+local funcs = helpers.funcs
+local rmdir = helpers.rmdir
+
+local file_prefix = 'Xtest-functional-ex_cmds-mkview_spec'
+
+describe(':mkview', function()
+  local tmp_file_base = file_prefix .. '-tmpfile'
+  local local_dir = file_prefix .. '.d'
+  local view_dir = file_prefix .. '.view.d'
+
+  before_each(function()
+    clear()
+    lfs.mkdir(view_dir)
+    lfs.mkdir(local_dir)
+  end)
+
+  after_each(function()
+    -- Remove any views created in the view directory
+    rmdir(view_dir)
+    lfs.rmdir(local_dir)
```

Comment:
Forgot to remove the `lfs`? :)

Reply:
No, either function would be fine fine. However it might actually be best to keep lfs in there since we are assuming the local_dir to be empty and lfs would error if it was not.

local_dir is only used for the ":lcd" option.